meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "Autometa" %}
+{% set data = load_setup_py_data() %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ data.get('version') }}
+
+source:
+  path: ./
+  # git_url: https://github.com/WiscEvan/Autometa.git
+  # git_rev: 2.0.0-cbld
```

Comment:
See other comments - link should be updated to main repo and version number should match to VERSION.

Reply:
I am applying a jinja template to pull the version from `setup.py`. Although pulling from git has been commented out and is not being used in the build process.

`git_rev = {{ data.get('version') }}`